.github/workflows/jekyll.yml

```diff
@@ -1,18 +1,60 @@
+
 name: Jekyll site CI
 
 on:
   pull_request:
     branches: [ master ]
 
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Build the site in the jekyll/builder container
-      run: |
-        docker run \
-        -v ${{ github.workspace }}:/srv/jekyll --env PAGES_REPO_NWO=${{github.repository}} -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
-        jekyll/builder:3.8.6 /bin/bash -c "gem install github-pages --no-document && chmod 777 /srv/jekyll && jekyll build"
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.127.0
+        with:
+          ruby-version: '3.1'
+          bundler-cache: true
+          cache-version: 0
+
+      - name: Setup Pages
+        id: pages
+        uses: actions/configure-pages@v2
+
+      - name: Set Jekyll environment
+        id: name
+        run: |
+          if [ "$REPO_OWNER" == "elixir-europe" ]; then
```

Comment:
Is this correct?

Reply:
@fbacall well spotted, is a sloppy copy paste mistake. To give context: In the future we would move with RDMkit to GitHub Actions for deployment instead of GitHub Pages. This Jekyll CI script would just be extended and the artifacts would get uploaded to the GitHub Pages environment. From that moment on we would display a banner when a deployment is ran other than on the ELIXIR Europe organisation in the shape of "this is a development instance ... ". 

But that is all for the future.

Reply:
Sounds good. Shall I go ahead and merge?

Reply:
sounds good to, in any case, I am standby for when things would look funky!